PYME/Acquire/HTTPSpooler.py

```diff
@@ -255,21 +255,23 @@ def StopSpool(self):
             for pt in self._pollThreads:
                 pt.join()
         
+        # save events and final metadata
+        # TODO - use a binary format for saving events - they can be quite
+        # numerous, and can trip the standard 1 s clusterIO.put_file timeout.
+        # Use long timeouts as a temporary hack because failing these can ruin
+        # a dataset
         if self._aggregate_h5:
-            clusterIO.put_file('__aggregate_h5/' + self.seriesName + '/final_metadata.json', self.md.to_JSON().encode(),
-                               serverfilter=self.clusterFilter)
-    
-            #save the acquisition events as json - TODO - consider a binary format as the events
-            #can be quite numerous
-            clusterIO.put_file('__aggregate_h5/' + self.seriesName + '/events.json', self.evtLogger.to_JSON().encode(),
-                               serverfilter=self.clusterFilter)
-        
+            clusterIO.put_file('__aggregate_h5/' + self.seriesName + '/final_metadata.json', 
+                               self.md.to_JSON().encode(), self.clusterFilter)
+            clusterIO.put_file('__aggregate_h5/' + self.seriesName + '/events.json', 
+                               self.evtLogger.to_JSON().encode(),
+                               self.clusterFilter, 10)
```

Comment:
```suggestion
                               self.clusterFilter, timeout=10)
```

Just to make it a bit more readable - the other parameters are mostly explained by the context